src/oncall/ui/static/js/oncall.js

```diff
@@ -2036,14 +2054,21 @@ var oncall = {
           url = this.data.url + oncall.data.user,
           data = $form.find('select[name="time_zone"]').val();
 
+      let userContactsElements = this.data.$form + ' input[type=text][name^="contactmode-"]';
```

Comment:
Lets merge these 2 let statements into the variable declaration above on line 2055.